KeyValuePair[] substitution
string instantiation
float64 confidence
float64 startTimestamp
float64 endTimestamp
